packages/shared/src/styles/media.ts

```diff
@@ -6,3 +6,4 @@ export const laptopL = `@media (min-width: 1360px)`;
 export const laptopXL = `@media (min-width: 1668px)`;
 export const desktop = `@media (min-width: 1976px)`;
 export const desktopL = `@media (min-width: 2156px)`;
+export const desktopXL = `@media (min-width: 2294px)`;
```

Comment:
Is this new breakpoint part of the design system?

Reply:
No, but the design system is broken. In figma the definition is: 2156 width should fit 7 columns. But if you multiply 7 columns with minimal width of 272px and add the column gaps of 32px and the padding of the container, you get to  `272px*7 + 32px*6 + 64px*2 = 2224px` which is more than the 2156px that the design system is talking about. 

2294 is basically just 2224 + 70px as a safety margin. This way, we keep the column size at at least 272px. It is similar to what we did with other breakpoints. E.g. for 6 columns, you only need `272px*6 + 32px*5 + 64px*2 = 1920px`, but the breakpoint is defined at 1976px instead.

Reply:
I see. That's a great catch. And I see that you already messaged the design team. Thank you for the fix! 🚀